tokio/src/runtime/enter.rs

```diff
@@ -104,6 +105,42 @@ cfg_blocking_impl! {
                 park.park()?;
             }
         }
+
+        /// Blocks the thread on the specified future, returning the value with
+        /// which that future completes.
+        pub(crate) fn block_on_timeout<F>(&mut self, mut f: F, timeout: Duration) -> Result<F::Output, ParkError>
+        where
+            F: std::future::Future,
+        {
+            use crate::park::{CachedParkThread, Park};
+            use std::pin::Pin;
+            use std::task::Context;
+            use std::task::Poll::Ready;
```

Comment:
minor style nit: some of these imports are now used by both `block_on` and `block_on_timeout`, should they be moved out of the individual functions?